lib/proforma/helpers/import_helpers.rb

```diff
@@ -94,7 +96,7 @@ def meta_data(any_data_node, use_namespace: false)
       private
 
       def convert_xml_node_to_json(any_node)
-        xml_snippet = Nokogiri::XML(any_node.to_xml(save_with: 0))
+        xml_snippet = Nokogiri::XML::DocumentFragment.parse any_node.to_xml(save_with: 0)
```

Comment:
Aren't we converting the `any_node` to a string here and parsing it again? I am not a fan of that, so what do you think about

```suggestion
        xml_snippet = Nokogiri::XML::DocumentFragment.new(Nokogiri::XML::Document.new, any_node)
```